astroquery/esa_hubble/core.py

```diff
@@ -0,0 +1,396 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+
+@author: Javier Duran
+@contact: javier.duran@sciops.esa.int
+
+European Space Astronomy Centre (ESAC)
+European Space Agency (ESA)
+
+Created on 13 Ago. 2018
+
+
+"""
+from astroquery.utils import commons
+from astropy import units
+from astropy.units import Quantity
+from astroquery.utils.tap.core import TapPlus
+from astroquery.utils.tap.model import modelutils
+from astroquery.query import BaseQuery
+
+from . import conf
+from astropy import log
+
+
+__all__ = ['EsaHubble', 'EsaHubbleClass']
+
+
+class ESAHubbleHandler(BaseQuery):
+
+    def __init__(self):
+        super(ESAHubbleHandler, self).__init__()
+
+    def get_file(self, filename, response, verbose=False):
+        file = open(filename, 'wb')
```

Comment:
Context managers are a better way to do this:
```suggestion
        with open(filename, 'wb') as file:
             file.write(response.content)
```